repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/valveSelfConfigure/MultiClusterMultiNodeTest.groovy

```diff
@@ -126,15 +99,5 @@ class MultiClusterMultiNodeTest extends Specification {
         then:
         thrown(ConnectException)
     }
-
-    def cleanup() {
```

Comment:
Absolutely. One of my complaints with our performance framework is that the lifecycle of certain things aren't as encapsulated as I would like. For example, `deproxy` has to be initialized in every test, but that declaration is in `ReposeValveTest` and the `cleanupSpec` block handles its shutdown. That's a bit tangential, though.

In this case, we are performing a `setup`, but expecting the `cleanupSpec` block to do the cleanup. So the lifecycle doesn't really match. By matching the lifecycle, I think we gain two things:

1. Clarity. `setupSpec` is cleaned up by `cleanupSpec`. So if I want to know what is handling the cleanup, if I had a `setupSpec`, I would first go look for a `cleanupSpec`.
2. Safety. This one is a bit iffy, because someone could change the `setupSpec` to `setup` and add more tests, but my hope is that by forcing the developer to make that change, the developer realizes that they will also need a `cleanup` block. Left as is, the developer could just add mores tests and things would appear to work. However, the old Repose instances wouldn't be shut down, only the newest instance would when `cleanupSpec` gets run.

If we wanted to match the lifecycle, we would either have to keep this `cleanup` block or change `setup` to `setupSpec`. This cleanup block is redundant code that the inherited `cleanupSpec` already contains, so to keep DRY, I would just change `setup` to `setupSpec`.

A lot of rationale for a tiny change, I know.